README.md

```diff
@@ -235,6 +235,8 @@ more information about the governance of the Node.js project, see
 * [joaocgreis](https://github.com/joaocgreis) -
 **João Reis** &lt;reis@janeasystems.com&gt;
 * [julianduque](https://github.com/julianduque) -
+**Josh Gavant** &lt;josh.gavant@outlook.com&gt;
+* [joshgav](https://github.com/joshgav) -
```

Comment:
You’re inserting yourself in the middle of julianduque’s entry… not sure that’s intended? ;)